### DISCUSSION TITLE
Load Mesh and Graph from Printer

### DISCUSSION THREAD

**Discussion by d42ohpaz** [jneilliii/OctoPrint-BedLevelVisualizer#461] [Q&A] [answered]
It is my understanding that the Update Mesh button will always create a new mesh and then load the graph from the resulting saved data - that works really well. However, I'm looking for how I would go about loading a saved mesh that was generated directly on the printer. I was able to create a command button that would load the mesh using the M420 GCode command, but how would I get the plugin to generate a new graph? Looking through the code, I'm lead the believe that the [afterTabChange](https://github.com/jneilliii/OctoPrint-BedLevelVisualizer/blob/427d6fd512ca58d84685d6517b1da6062fddc5eb/octoprint_bedlevelvisualizer/static/js/bedlevelvisualizer.js#L366-L368) event should load draw the mesh, but it doesn't seem to do that (for me). 

Am I missing something? Thank you!

**Comment by jneilliii** [comment, marked as answer]
Just use M420 V in the settings for update mesh and that will just grab from EEPROM.

**Reply by hayden-t** [reply]
thank you for this !!
